lessons/ro/chapter_5.yaml

```
@@ -0,0 +1,223 @@
+- title: Capitolul 5 - Proprietatea și împrumutul Datelor
+  content_markdown: >
+    Rust are o paradigmă unică pentru gestionarea memoriei în comparație cu alte limbaje
+    de programare. O să analizăm
+
+    pe rând comportamentul și verificările compilatorului pentru a nu deveni
+    copleșitor. Este important să rețineți faptul că
+
+    la finalul zilei, regulile prezentate nu există pentru a vă face viața grea, dar ca să vă ajute
+    să scrieți cod mai puțin predispus la erori!
+- title: Proprietate
+  content_markdown: >
+    Instanțiind un tip și **legându-l** de un nume de variabilă vom crea o resursă în memorie
+    pe care compilatorul de Rust o va valida
+    
+    pe tot parcursul **duratei sale de viață**. Variabila de care a fost legat se numește
+    **proprietarul** resursei.
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=struct%20Foo%20%7B%0A%20%20%20%20x%3A%20i32%2C%0A%7D%0A%0Afn%20main()%20%7B%0A%20%20%20%20%2F%2F%20Instan%C8%9Biem%20struct-uri%20%C8%99i%20leg%C4%83m%20de%20variabile%0A%20%20%20%20%2F%2F%20pentru%20a%20crea%20resurse%20%C3%AEn%20memorie%0A%20%20%20%20let%20foo%20%3D%20Foo%20%7B%20x%3A%2042%20%7D%3B%0A%20%20%20%20%2F%2F%20foo%20este%20proprietarul%0A%7D%0A
+- title: Gestionarea resurselor bazată pe scope
+  content_markdown: |
+    Rust folosește finalul unui scope ca moment pentru a destructura și a dealoca o resursă.
```

```suggestion
    Rust folosește finalul unui domeniu de vizibilitate ca moment pentru a destructura și a dealoca o resursă.
```

*Scope* is *domeniu de vizibilitate*.